tests/component-tests/textToSpeech/js/TextToSpeechTests.js

```diff
@@ -11,13 +11,40 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 */
 
 // Declare dependencies
-/* global fluid, jqUnit, speechSynthesis */
+/* global fluid, jqUnit, speechSynthesis, SpeechSynthesisUtterance */
 
 (function () {
     "use strict";
 
     fluid.registerNamespace("fluid.tests");
 
+    // Ensures that TTS is supported in the browser, including cases wehre the
+    // feature is detected, but where the underlying audio engine is missing.
+    // For example VM's on SauceLabs.
+    //
+    // returns a promise
```

Comment:
I've only included this check in the tests because it runs asynchronously. All of the other standard progressive enhancement checks are all synchronous, so introducing this inline with those could/would cause confusion for the time that they would resolve. So I'm not sure what you mean by supported. I believe it is useful to the tests, but not necessarily for general use. What do you think?